### DISCUSSION TITLE
Web profile colours

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#28816] [Ideas] [answered]
I really like the feature of changing the profile colour with supporter, but it could get a bit better.

Basically, the best, and the hardest to execute would be making the colour select work similarly to https://paletton.com this could also allow to make the colour gradient, and in specific "mode"

The also good option, would be making it similar, but without gradients and just singular colours, but you could change the contrast, opacity and/or some other stuff

One of the most important things here would be an ability to to paste an RGB or some other colour "ID" so you can get the exact colour you want

How to place it? Make a button that basically rolls the colour select out in the same place the current colour select is, or add it to profile settings and add a "preview" there, so you can still see how it would look, but it has it own space, and isn't going to take much of your screen on the profile page

This is probably already worked on, but I decided to post it here just on case it wasn't

**Comment by GSculerlor** [comment, marked as answer]
should be on https://github.com/ppy/osu-web ?

**Reply by Tenexxt** [reply]
I'm new to GitHub, so I have a hard time navigating, sorry! Tho, am I supposed to close this post or can it just stay here? Because I have no idea
